PROJECT

```diff
@@ -25,6 +25,15 @@ resources:
   kind: OCSInitialization
   path: github.com/red-hat-storage/ocs-operator/api/v4/v1
   version: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: openshift.io
+  group: ocs
+  kind: StorageAutoScaler
+  path: github.com/red-hat-storage/ocs-operator/api/v4/v1
```

Comment:
@parth-gr Just to confirm: is `/api/v4` intended?

Reply:
yaa as the repo is github.com/red-hat-storage/ocs-operator/v4

Reply:
This is because it is defined this way in go.mod